ansible/roles/tileserver_step2/tasks/files/settings.js

```diff
@@ -14,5 +14,6 @@ var overlays = {
     'maxspeed' : L.tileLayer('/maxspeed/{z}/{x}/{y}.png', {maxZoom: maxZoom, attribution: 'Map data &copy; <a href="http://openstreetmap.org">OpenStreetMap</a> contributors, rendering CC-BY-SA OpenRailwayMap'}),
     'signals' : L.tileLayer('/signals/{z}/{x}/{y}.png', {maxZoom: maxZoom, attribution: 'Map data &copy; <a href="http://openstreetmap.org">OpenStreetMap</a> contributors, rendering CC-BY-SA OpenRailwayMap'}),
     'electrification' : L.tileLayer('/electrification/{z}/{x}/{y}.png', {maxZoom: maxZoom, attribution: 'Map data &copy; <a href="http://openstreetmap.org">OpenStreetMap</a> contributors, rendering CC-BY-SA OpenRailwayMap'}),
+    'gauge' : L.tileLayer('/gauge/{z}/{x}/{y}.png', {maxZoom: maxZoom, attribution: 'Map data &copy; <a href="http://openstreetmap.org">OpenStreetMap</a> contributors, rendering CC-BY-SA OpenRailwayMap'}),
```

Comment:
This becomes ugly. Can the option list be assigned to an explicit variable before and just referenced in these instances?

Reply:
I templated it via ansible now (changing "infrastructure" also to "standard" in line 13.) and also added the loop in `tiles.openrailwaymap.org.inc`.